contents/site/meta/wiki_sunset.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "diaspora* Wiki Sunset"
+subtitle: The information you requested got relocated.
+---
+
+<%=
+  partial "components/inline_image",
+  alt: "We are vewwy sowwy.",
+  src: static_url("site/mr_abdoun_the_scottish_cat.jpg")
+%>
+
+In Spring 2019, we decided to discontinue the diaspora\* Project Wiki and instead merge the information with the project site and the dedicated installation guides. This allows us easier control and maintenance of the project's infrastructure, as well as streamlining contributions to both the wiki and our project sites. While we attempted to keep the old URLs working by setting up redirects, we could not redirect all wiki links. If you are reading this, it means you followed a link to an old wiki resource that we could not reallocate directly. Sorry about that.
+
+To help you find what you are looking for, here is a list of resources available to you:
+
+* If you need help using diaspora\*, maintaining a pod or have development related questions, check out our [guides][guides].
+* If you are interested in general information on what diaspora\* is and why we are doing what we do, check out the [project website][project-site].
+* If you want to install your own pod and are looking for instructions, check out our [installation guides][install].
+
```

Comment:
How about a separate line with a link to a list of page/article titles on the new site – like the existing wiki index? That way a person might find the nearest equivalent to what they were looking for.

Reply:
Hm, no. A sitemap would actually be giant. But your question made me realized we need https://github.com/diaspora/north-star/issues/20, so let's do that.